README.md

```diff
@@ -1 +1,18 @@
 # Task-Tracker-App
+
+The app is designed to work with the [Task-Tracker-Device](https://github.com/Task-Tracker-Systems/Task-Tracker-Device).
+
+## Build
+
+### Local
+Run the gradle build in each directory or use the `build.sh` file.
+
+### Docker
+Powershell
+```Powershell
+docker build -t task-tracker-systems/task-tracker-app:1 .
+docker run --rm -v ${PWD}:/dev/mnt task-tracker-systems/task-tracker-app:1
```

Comment:
FWI: This took about 14mins the first time. Around 8mins the second time.

This is quite slow...

Reply:
I would call it too slows. However, it would not be my main priority to improve the performance here